test/onigumo_test.exs

```diff
@@ -39,27 +25,18 @@ defmodule OnigumoTest do
   end
 
   @tag :tmp_dir
-  test("load a single URL from a file", %{tmp_dir: tmp_dir}) do
```

Comment:
so we dont test a URL, because it is enough to test multiple URLs is that right?

Reply:
You are right, but it was my mistake to include the change in this pull request. I mistakenly pasted modified tests from #142. My intention was only to change the order of the tests, not to change them. I reverted the tests to their shape from “master”.

Nice catch, thank you!